.github/scripts/codecheck.py

```diff
@@ -83,6 +94,74 @@ def check_workspace_and_package_versions_equal():
 
     return result
 
+def internal_check_dependency_versions(root_node, dependencies_name: str, file_path) -> bool:
+    '''
+    Both dependencies and dev-dependencies have the same structure, so we check the versions the same way for both
+    root_node is the root node of the Cargo.toml file
+    file_path is the path of the Cargo.toml file, for logging
+    '''
+
+    res = True
+
+    # list of crates, whose version can have build version
+    exempted_crates = [
+       # 'ctor'    # left here as an example, remove if you ever add one crated that is exempted
+    ]
+
+    if dependencies_name in root_node:
+        deps = root_node[dependencies_name]
+        for dep in deps:
+            # skip exempted crates
```

Comment:
I'm not convinced by exempted here (or anywhere in this) but I'm struggling to care too much

Reply:
I think both are correct, one is passive and the other is an adjective.